bindings/matrix-sdk-ffi/src/authentication_service.rs

```diff
@@ -91,6 +92,13 @@ pub enum AuthenticationError {
     #[error("An error occurred with OIDC: {message}")]
     OidcError { message: String },
 
+    #[error("The supplied callback URL used to complete SSO is invalid.")]
+    SsoCallbackUrlInvalid,
+    #[error("The supplied callback URL used to complete SSO does not contain a login token.")]
+    SsoCallbackUrlMissingLoginToken,
+    #[error("Logging in with the token from the supplied callback URL failed.")]
+    SsoLoginWithTokenFailed,
```

Comment:
```suggestion
    #[error("The supplied callback URL used to complete SSO is invalid.")]
    SsoCallbackUrlInvalid,

    #[error("The supplied callback URL used to complete SSO does not contain a login token.")]
    SsoCallbackUrlMissingLoginToken,

    #[error("Logging in with the token from the supplied callback URL failed.")]
    SsoLoginWithTokenFailed,
```

Reply:
I personally don't mind the format but the OIDC errors above seem to not use a separating blank line either. Should I change it there as well?